BREAKING_CHANGES.md

```diff
@@ -4,6 +4,19 @@ ChameleonForms Breaking Changes
 Version 2.0.0
 =============
 
+Deprecated `WithoutLabel` method on `IFieldConfiguration`. It still works (for now), but the method has been marked with the `[Obsolete]` attribute.

```

Comment:
It would be nice to get my other WIP PR merged as well, but not sure when I'll have time to finish it.